### ISSUE TITLE
Initial Gamma not Respected

### ISSUE THREAD

**Issue by MosGeo** [napari/napari#6715]
### 🐛 Bug Report

Initial gamma values are not respected in the viewer. It is shown in the slider correctly but it is not applied in the viewer. Interestingly, it is applied in the thumbnail preview.

### 💡 Steps to Reproduce

Run the following in the napari console. You will note that the gamma slider has changed but it is not applied on the image.
```python
from skimage import data
viewer.add_image(data.cell(), name='cell', gamma=.5)
````

### 💡 Expected Behavior

Gamma values are respected in the viewer

### 🌎 Environment

napari: latest in pypi (4.19)
OS: Windows 10 and 11, 64bit
Python: 3.10 (64bit)

### 💡 Additional Context

A quick workaround is also welcome for the time being :)

**Comment by Czaki** [comment]
It looks solved in #6650 but not released yet.

**Comment by MosGeo** [comment]
I can confirm that it is solved in #6650. Thanks. Closing.
